gulpfile.js

```diff
@@ -1,12 +1,14 @@
-var gulp = require('gulp');
-var concat = require('gulp-concat');
+var gulp = require('gulp'),
+    concat = require('gulp-concat'),
+    livereload = require('gulp-livereload');
 
 gulp.task('watch', function () {
 	gulp.watch(['./public/js/*.js', './index.html'], function () {
-    reload();
+    livereload.reload();
```

Comment:
Should find out the reason why it is throwing the error rather than just using something different and pushing to master because it forces everyone else to suddenly use something different which may or may not work for them.